.github/workflows/build.yml

```diff
@@ -70,12 +58,26 @@ jobs:
     strategy:
       matrix:
         include:
-          - {goos: "linux", goarch: "arm"}
-          - {goos: "linux", goarch: "arm64"}
-          - {goos: "linux", goarch: "386"}
-          - {goos: "linux", goarch: "amd64"}
-          - {goos: "linux", goarch: "amd64", gotags: "fips", env: "CGO_ENABLED=1 GOEXPERIMENT=boringcrypto", fips: "+fips1402", pkg_suffix: "-fips" }
-          - {goos: "linux", goarch: "arm64", gotags: "fips", env: "CGO_ENABLED=1 GOEXPERIMENT=boringcrypto CC=aarch64-linux-gnu-gcc", fips: "+fips1402", pkg_suffix: "-fips" }
+          - { goos: "linux", goarch: "arm" }
+          - { goos: "linux", goarch: "arm64" }
+          - { goos: "linux", goarch: "386" }
+          - { goos: "linux", goarch: "amd64" }
+          - {
+              goos: "linux",
+              goarch: "amd64",
+              gotags: "fips",
+              env: "CGO_ENABLED=1 GOEXPERIMENT=boringcrypto",
+              fips: "+fips1402",
+              pkg_suffix: "-fips",
+            }
+          - {
+              goos: "linux",
+              goarch: "arm64",
+              gotags: "fips",
+              env: "CGO_ENABLED=1 GOEXPERIMENT=boringcrypto CC=aarch64-linux-gnu-gcc",
+              fips: "+fips1402",
+              pkg_suffix: "-fips",
+            }
```

Comment:
We might not want to reformat in these backports bc it'll bork any future backports to these lines (though I'm down to reformat from `main` if you want)